shapes/produce_shapes_2017.py

```diff
@@ -1532,5 +1541,5 @@ def main(args):
 
 if __name__ == "__main__":
     args = parse_arguments()
-    setup_logging("{}_produce_shapes.log".format(args.tag), logging.INFO)
+    setup_logging("output/log/{}_{}_{}_shapes.log".format(args.era, args.train_method, ",".join(args.channels)), logging.INFO)
```

Comment:
The folders output/... are not there in a fresh installation, so the shape production will always crash at first - so ideally they should automatically be created in the first production

Reply:
Make sure that the scripts produce_shapes_2017.py, produce_shapes_2018.py and  produce_shapes_2016.py have consistent options, as we call them with the same .sh script

Reply:
> The folders output/... are not there in a fresh installation, so the shape production will always crash at first - so ideally they should automatically be created in the first production
moved it to a method in utils/bashFunctionCollection.sh that i call from the run_analysis